docs/reference/generated/menu-positioner.json

```diff
@@ -2,6 +2,10 @@
   "name": "MenuPositioner",
   "description": "Positions the menu popup against the trigger.\nRenders a `<div>` element.",
   "props": {
+    "collisionAvoidance": {
+      "type": "CollisionAvoidance",
+      "description": "Determines how to handle collisions when positioning the popup."
+    },
```

Comment:
The type is too big to display in our current table design, but will work with the new Accordion-style one